Equilobe.DailyReport.SL/PolicyService.cs

```diff
@@ -92,5 +87,35 @@ public static PolicyBuffer SyncPolicy(JiraPolicy jiraPolicy, PolicySummary polic
 
             return policyBuffer;
         }
+
+        public static PolicyBuffer GetPolicyBufferFromDb(string uniqueProjectKey)
```

Comment:
PolicySummary : IReportSetting
PolicyDetails : ISerializedPolicy
PolicyBuffer : IReportSetting, ISerializedPolicy

deci PolicyBuffer e clasa care face "combinatii" intre DB si UI in pagina de details [de asemenea updateaza si reportTime si serializedPolicy) - cool stuff